infrastructure/kube/keep-test/electrumx-statefulset.yaml

```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: electrumx
+  namespace: default
+  labels:
+    app: bitcoin
+    type: electrumx
+spec:
+  replicas: 1
+  serviceName: electrumx
+  volumeClaimTemplates:
+  - metadata:
+      name: electrumx-data
+    spec:
+      storageClassName: electrumx
+      accessModes: [ReadWriteOnce]
+      resources:
+        requests:
+          storage: 30Gi
+  selector:
+    matchLabels:
+      app: bitcoin
+      type: electrumx
+  template:
+    metadata:
+      labels:
+        app: bitcoin
+        type: electrumx
+    spec:
+      containers:
+      - name: electrumx-server
+        image: lukechilds/electrumx:v1.16.0
+        imagePullPolicy: Always
+        ports:
+          - name: tcp
+            containerPort: 50001
+          - name: ssl
+            containerPort: 50002
+          - name: ws
+            containerPort: 50003
+          - name: wss
+            containerPort: 50004
+          - name: rpc
+            containerPort: 8000
```

Comment:
Can we align it with prod configuration and use the same ports as in the service?
```
        ports:
          - containerPort: 443
          - containerPort: 8080
          - containerPort: 8443
          - containerPort: 8000
```